spec/spec_helper.rb

```diff
@@ -31,6 +31,6 @@
 
 Rspec.configure do |config|
   config.after(:each) do
-    FileUtils.rm ['test.db'], force: true
+    FileUtils.rm ["#{ENV['MUMUKI_DAYBREAK_NAME']}.db"], force: true
```

Comment:
Please provide a default when the ENV is not set. Otherwise you may remove something else.